clickable

```diff
@@ -1093,7 +1099,35 @@ class PythonClickable(PureClickable):
     # The only difference between this and the Pure template is that this doesn't force the "all" arch
     pass
 
+class GoClickable(Clickable):
+    def go_install(self):
```

Comment:
Any reason this isn't just in the _build() function?

Reply:
Changed this